tests/rules/class-name-format.js

```diff
@@ -0,0 +1,231 @@
+'use strict';
+
+var lint = require('./_lint');
+
```

Comment:
Can we add the SCSS and Sass block comments as with the other rules here just to give a better visual separation.

https://github.com/sasstools/sass-lint/blob/develop/tests/rules/final-newline.js#L5